### PR TITLE
adding code

### PR DESCRIPTION
**Pull request by rangadevops1993**
please review the code

### CODE REVIEW

File: ansible.txt.txt [hunk 1]
```diff
@@ -4,4 +4,4 @@ ansible playbook for java package
 - hosts: localhost
   tasks:
 -name: install java
-       - java
\ No newline at end of file
+       - maven
```
Comment (sivaazure1999) [inline ansible.txt.txt:7]: add maven version

File: terraform.txt [hunk 2]
```diff
@@ -1,5 +1,5 @@
 awsresource "aws_instance" "demo-server"{
 key_pair = "centos"
 ami = "ami-8k60ke53jde8tbn"
-instance_type = "t2_nano"
+instance_type = "t2_micro"
```
Comment (sivaazure1999) [inline terraform.txt:4]: change instance type to t2.large

**Review by sivaazure1999 — COMMENTED** [review]
added review comments please work

File: terraform.txt (hunk 2 above)
Comment (rangadevops1993) [reply terraform.txt:4]: changed t2 large

File: ansible.txt.txt (hunk 1 above)
Comment (rangadevops1993) [reply ansible.txt.txt:7]: added maven version

**Review by sivaazure1999 — APPROVED** [review]
looks good

**Pull request merged**